src/Api/Controller/TokenController.php

```diff
@@ -38,8 +41,14 @@ class TokenController
      */
     private $jsonDecode;
 
+    /**
+     * @var ContainerInterface
+     */
+    private $container;
+
     public function __construct(
         Payum $payum,
+        ContainerInterface $container,
```

Comment:
PaymentStorage service is autowireable. In general, try to avoid container injection as much as possible.